script/Arcueid/Day 10/10_00_ARC10_5D.txt

```diff
@@ -1039,12 +1040,12 @@ My sense of reason crumbles.
 [sha:cbc7f301b48a3ae4ab8b6d9b9b37f70c476ee414]{
 -- Page 619, Offset 25912.
 -- それは、根源的な恐怖だった。
-It was a primordial fear.
+It is a primordial fear.
```

Comment:
I think `was` because he's referring to the instant of terror he has seen, past?